src/vs/workbench/contrib/testing/browser/testingDecorations.ts

```diff
@@ -388,34 +414,18 @@ export class TestingDecorations extends Disposable implements IEditorContributio
 
 		this.attachModel(editor.getModel()?.uri);
 		this._register(decorations.onDidChange(() => {
-			if (this.currentUri) {
-				decorations.syncDecorations(this.currentUri);
+			if (this._currentUri) {
+				decorations.syncDecorations(this._currentUri);
 			}
 		}));
 		this._register(this.editor.onDidChangeModel(e => this.attachModel(e.newModelUrl || undefined)));
-		this._register(this.editor.onMouseDown(e => {
-			if (e.target.position && this.currentUri) {
-				const modelDecorations = editor.getModel()?.getDecorationsInRange(Range.fromPositions(e.target.position)) ?? [];
-				if (!modelDecorations.length) {
-					return;
-				}
-
-				const cache = decorations.syncDecorations(this.currentUri);
-				for (const { id } of modelDecorations) {
-					if ((cache.getById(id) as ITestDecoration | undefined)?.click(e)) {
```

Comment:
We still need to call the .click() handler of decorations to make normal left-clicks work, so most of this code block needs to remain (though we can have a method to reduce duplication)

Reply:
I don't think I needed to delete this actually--removed it while I was trying to figure out why the context menu wasn't showing up (because of `e.preventDefault`).